go.mod

```diff
@@ -18,12 +18,13 @@ require (
 require (
 	dario.cat/mergo v1.0.0 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
-	github.com/ProtonMail/go-crypto v0.0.0-20230717121422-5aa5874ade95 // indirect
+	github.com/ProtonMail/go-crypto v0.0.0-20230828082145-3c4c8a2d2371 // indirect
 	github.com/acomagu/bufpipe v1.0.4 // indirect
 	github.com/cloudflare/circl v1.3.3 // indirect
+	github.com/cyphar/filepath-securejoin v0.2.4 // indirect
```

Comment:
@cpanato Do you know why this new dependency was introduced?

Reply:
It has been added with this commit: https://github.com/go-git/go-git/commit/cf3a75c4233dc5694a209a336daf71b8171f1013

Reply:
Yes. But this particular module was present in earlier versions also. Just that a replace directive was used in place of it. Ref: https://github.com/go-git/go-billy/commit/e223a6609192f18b76a57b31a9a0338515a21789. I am confused why it was not present earlier in the go.mod

Anyway it doesnt block the PR.